### PR TITLE
Feature/extend icub25to icub_feet_fixed and icub_standing

### PR DESCRIPTION
**Pull request by nunoguedelha**
- Renamed the model `icub_fixed_no_hands` to `icub_fixed` since the base URDF model (autogenerated from CAD) might or might not have the hands parts.
- Updated the model `icub_feet_fixed` for handling the `ROBOT_NAME` parameter and created `model.config.in`.
- created model `icub_standing` for **walking** and **Yoga** demos.

### CODE REVIEW

File: models/icub_feet_fixed/icub_feet_fixed.sdf [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <?xml version='1.0'?>
-<sdf version='1.4'>
-  <model name="iCub">
+<sdf version='1.5'>
+  <model name="iCub_feet_fixed">
     <include>
-      <uri>model://icub</uri>
-      <pose>0.0 0 0.6 0 0 3.14</pose>
+      <uri>model://@ROBOT_NAME@</uri>
```
Comment (traversaro) [inline models/icub_feet_fixed/icub_feet_fixed.sdf:5]: For clarity, I would append a `.in` at the end of the name of any file that needs to be configured by CMake's `configure_file` . What do you think @nunoguedelha ?

**Review by traversaro — COMMENTED** [review]
Fine for me for the most part, but the icub standing model will be spawned with the usual home position right? Edit: I did not understand what the icub standing was supposed to do.

**Comment by traversaro** [comment]
If I understand correctly, the icub standing will be necessary to work with the new automatically generated models. As such, I would consider migrating it directly to `icub-models`, as otherwise the model will be unusable.

**Comment by diegoferigo** [comment]
> If I understand correctly, the icub standing will be necessary to work with the new automatically generated models. As such, I would consider migrating it directly to icub-models, as otherwise the model will be unusable.

Next week with also @gabrielenava we're going to discuss a new repository structure for the the icub models (sdf, urdf, ini, configs, ...). I'm rather to keep raw models in `icub-models`, and model's poses / configuration / worlds somewhere else. In this case, gazebo would need to only have the path of the poses.

**Comment by diegoferigo** [comment]
I agree on appending `.in` to the sdf files (also for the other poses).

**Review by diegoferigo — APPROVED** [review]

File: models/icub_feet_fixed/icub_feet_fixed.sdf (hunk 1 above)
Comment (nunoguedelha) [reply models/icub_feet_fixed/icub_feet_fixed.sdf:5]: Fully agree. I will push an update right away.

Comment (nunoguedelha) [reply models/icub_feet_fixed/icub_feet_fixed.sdf:5]: Regarding the home position...
Each model `icub_fixed`, `icub_feet_fixed`, `icub_standing` will be spawned with the respective home position (change in a next branch).
Actually `icub_standing` is supposed to be used for the Yoga and walking demos. I can replace it by 2 models: `icub_standing_yoga` (or just `icub_yoga`) and `icub_walking`.

**Comment by nunoguedelha** [comment]
> Next week with also @gabrielenava we're going to discuss a new repository structure for the the icub models (sdf, urdf, ini, configs, ...). I'm rather to keep raw models in icub-models, and model's poses / configuration / worlds somewhere else. In this case, gazebo would need to only have the path of the poses.

The idea indeed is that each model `icub_fixed`, `icub_feet_fixed`, `icub_standing` includes the raw model and a pose configuration.
@traversaro This is still compatible with having a raw model that can be inserted directly in the Gazebo world, floating. But inserting directly a raw model in Gazebo still poses a problem for the path of the inertial sensors as per the current handling done in the respective gazebo-yarp plugins (`gazebo-yarp-plugins/plugins/inertialmtb/`,`gazebo-yarp-plugins/plugins/inertialmtbPart/`,`libgazebo_yarp_distributedinertials.so`).
@traversaro regarding the idea we discussed last week on having the composed models `icub_fixed`,`icub_on_seesaw`, ... defined as `worlds` only, and not `models`, I would rather drop it since I realized that people are more comfortable using the models instead of the worlds... and it is actually more flexible.

**Comment by diegoferigo** [comment]
Am I the only one who likes worlds? :sweat_smile:

**Pull request merged**